srcpkgs/OpenCoarrays/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'OpenCoarrays'
+pkgname=OpenCoarrays
+version=2.9.2
+revision=1
+build_style=cmake
+short_desc="OpenCoarray fortran PAGS paralellization library"
+maintainer="Alberto Ramos <alberto.ramos@ific.uv.es>"
+hostdepends="gcc gcc-fortran openmpi openmpi-devel cmake"
```

Comment:
`hostmakedepends`, -devel things should go in `makedepends`. The `build_style` is `cmake`, no need to include `cmake` in `hostmakedepends` or in `makedepends`. Looking at the project, you might want `openmpi-devel` for the development files in `makedepends`.